src/main/java/org/cbioportal/service/impl/AlterationCountServiceImpl.java

```diff
@@ -223,6 +263,197 @@ public Pair<List<CopyNumberCountByGene>, Long> getPatientCnaGeneCounts(List<Mole
         );
     }
 
+    @Override
```

Comment:
@haynescd what's all this stuff?

Reply:
This is the stuff for the new clickhouse implementation